src/allmydata/frontends/magic_folder.py

```diff
@@ -33,6 +33,8 @@ def get_inotify_module():
             from allmydata.windows import inotify
         elif runtime.platform.supportsINotify():
             from twisted.internet import inotify
+        elif sys.platform != "linux":
+            from allmydata.watchdog import inotify
```

Comment:
Ah, watchdog falls back to a polling implementation that should work (inefficiently) on all platforms supported by Python. So this can just be:

```
else:
    from allmydata.watchdog import inotify
```